FedAvgExample.py

```diff
@@ -3,14 +3,16 @@
 from Algorithms.FedAvg import FedAvg
 from Models.Callbacks.Callbacks import Callbacks
 import matplotlib.pyplot as plt
+import torch
 
 number_of_clients = 5
 batch_size = 16
 dataloader = Dataloader(number_of_clients)
 test_data = dataloader.get_test_dataloader(batch_size)
+device = torch.cuda.current_device() if torch.cuda.is_available() else 'cpu'
```

Comment:
Orsakar inte detta en bugg om vi kör på en maskin utan CUDA-support? Borde inte default vara None?

Reply:
Kan ju använda 'cpu' men då får vi ändra default-argumentet och alla if-satser.

Reply:
Hmm kanske, ´torch.cuda.is_available()´ kanske bara funkar om du redan har drivers installerade? Samtidigt är ju cudatools en grej i får miljö, så kanske att det funkar ändå? Funkar det att köra som det är för dig?

´tensor.to('cpu')´ är default sätt att allokera en tensor till cpu, därför jag har skrivit 'cpu' här. None använder vi ju för att checka om vi vill ändra device över huvudtaget, men poängen kanske är att i vårt standard start skript borde device vara None?